### ISSUE TITLE
Indexing a vector measure does not work after date_shift on vector 

### ISSUE THREAD

**Issue by mingkongwoo** [atoti/atoti#111]
Refer to the attached example.
[date_shift.zip](https://github.com/atoti/atoti/files/4936624/date_shift.zip)

## Steps to reproduce
I loaded a store containing vectors of 4 numeric values.
I do a `date_shift` on the vector measure.
```
m["MtM at Index"] = m["MtM.SUM"][m["Index"]]

m["Shifted MtM"] = tt.date_shift(
    m["MtM.SUM"], on=h["BizDate"], offset="-1D", method="interpolate"
)
m["Shifted MtM at Index"] = m["Shifted MtM"][m["Index"]]
```

## Actual Result
When I query the Shifted MtM at Index measure, only the first index is returned.
![image](https://user-images.githubusercontent.com/30063740/87766598-febb1f80-c84b-11ea-8f59-2253a8d6feb0.png)

## Expected Result
I should be able to see values in the vector for other indices.

## Environment
- atoti: 0.4.2
- Python: 3.7
- Operating System: Windows

**Comment by fabiencelier** [comment]
Looks like the issue is with the date_shift and the analysis hierarchy:

![shift issue](https://user-images.githubusercontent.com/17087544/87773320-47121780-c823-11ea-9dcc-a0cfddcc595c.png)

**Comment by bloussou** [comment]
This is fixed in version `0.5.1` :
![image](https://user-images.githubusercontent.com/32448554/106606900-f1a11c00-6562-11eb-9e35-14a21b963851.png)
